desc/compute/_bootstrap.py

```diff
@@ -186,17 +195,15 @@ def j_dot_B_Redl(geom_data, profile_data, helicity_N=None):
 
     # Redl eq (14):
     X32e = f_t / (
-        (
-            1
-            + 0.23 * (1 - 0.96 * f_t) * jnp.sqrt(nu_e) / jnp.sqrt(Zeff)
-            + 0.13
-            * (1 - 0.38 * f_t)
-            * nu_e
-            / (Zeff * Zeff)
-            * (
-                jnp.sqrt(1 + 2 * jnp.sqrt(Zeff - 1))
-                + f_t * f_t * jnp.sqrt((0.075 + 0.25 * (Zeff - 1) ** 2) * nu_e)
-            )
+        1
+        + 0.23 * (1 - 0.96 * f_t) * jnp.sqrt(nu_e) / jnp.sqrt(Zeff)
+        + 0.13
+        * (1 - 0.38 * f_t)
+        * nu_e
+        / (Zeff * Zeff)
+        * (
+            jnp.sqrt(1 + 2 * jnp.sqrt(Zeff - 1))
+            + f_t * f_t * jnp.sqrt((0.075 + 0.25 * (Zeff - 1) ** 2) * nu_e)
```

Comment:
Some of these refactoring are all flake8's doing, which only modifies files when the file to commit has been modified.